content/guides/pricing-concepts.md

```diff
@@ -0,0 +1,120 @@
+
+In that case, you need to calculate the price based on BookingSync data. Depending on the synchronous (live quote request) or asynchronous (cached rates) nature of your application's flow, there are few ways to do it:
+
+### 1. Live Quote request (recommended for synchronous flow)
+
+Live Quote is the best way to get the exact price for given rental for specified dates as it operates on the data that is always up-to-date, not a cached one, and also offers a precise breakdown of the way how the price has been calculated. To take advantage of this feature, you just need to send a request to [`/rentals/search` endpoint](http://developers.bookingsync.com/reference/endpoints/rentals/#search-rentals) with the following params:
+
+* `rental_id` (required) - BookingSync id of the rental you want to create the booking for
+* `start_at` (required) - the start date of the booking
+* `end_at` (required) - the end date of the booking
+* `adults` (required) - the number of adults for the booking
+* `children` (optional) - the number of children for the booking
+
+If the rental is available for the provided dates, you will get a non-empty response body looking like this:
+
+~~~json
+{
+  "rentals": [
+    {
+      "id": 1,
+      "currency": "EUR",
+      "damage_deposit": 35.5,
+      "final_price": 858.33,
+      "initial_price": 700.0,
+      "price_details": {
+        "taxes": [
+          {
+            "name": {
+              "en": "VAT"
+            },
+            "percentage": "10.0",
+            "amount": "58.33",
+            "taxable_type": "Rental",
+            "taxable_id": 2,
+            "tax_id": 1,
+            "included": false
+          }
+        ],
+        "fees": [
+          {
+            "name": {
+              "en": "some fee"
+            },
+            "required": true,
+            "price": "100.0",
+            "quantity": 1,
+            "id": 10,
+            "rentals_fee_id": 12
+          }
+        ]
+      },
+      "price_to_pay_now": 858.33,
+      "taxes": [
+        {
+          "name": {
+            "en": "VAT"
+          },
+          "percentage": "10.0",
+          "amount": "58.33"
+        }
+      ],
+      "updated_at": "2014-01-21T11:46:15Z",
+      "links": {
+        "rentals_fees": "http://www.bookingsync.com/api/v3/rentals_fees/{rentals.rentals_fees}"
+      }
+    }
+  ]
+}
+~~~
+
+If the response contains any `taxes` or `fees` inside `price_details`, you MUST include them in the request body when [creating a booking](http://developers.bookingsync.com/reference/endpoints/bookings/#create-a-new-booking).
+
+Keep in mind that `taxes` outside `price_details` are just grouped summed taxes of the same kind that should be used only for presentation purposes.
+
+The response will contain three top-level attributes that are related to the price:
+
+* `initial_price` - the initial price (without taxes and fees). If you don't apply any discounts, this will be equal to the price for only the rental.
```

Comment:
`without taxes` > `without excluded taxes`. I think explaining the difference between included and excluded taxes is needed then.

Reply:
Might be better to do it in a separate PR, I think this is good enough to be live and we could polish it after current priorities.